src/Simplex/Messaging/Agent.hs

```diff
@@ -626,7 +651,7 @@ getConnectionMessage' :: AgentMonad m => AgentClient -> ConnId -> m (Maybe SMPMs
 getConnectionMessage' c connId = do
   whenM (atomically $ hasActiveSubscription c connId) . throwError $ CMD PROHIBITED
   withStore c (`getConn` connId) >>= \case
-    SomeConn _ (DuplexConnection _ rq _) -> getQueueMessage c rq
+    SomeConn _ (DuplexConnection _ (rq :| _) _) -> getQueueMessage c rq
```

Comment:
should it be from all active queues? - maybe leave a comment.. though if notification comes it's probably for the leading